### DISCUSSION TITLE
[Uczymy się architektury] 🚀🚀 Architektura w Azure - AAD część 2 #azure-solutions-architect

### DISCUSSION THREAD

**Discussion by aryll** [AzureCommunityPL/group#165] [Wydarzenia]
Cześć! 😃

Już wkrótce widzimy się znowu podczas drugiego podejścia do architektury z użyciem **Azure Active Directory**!

Uczymy się jak **włączyć AAD do naszego rozwiązania** implementując takie zadanie:
Połącz w bezpieczny sposób aplikację web z bazą danych (np. Cosmos DB) i pobierz informacje korzystając z tokenu użytkownika.

Elementy które warto wziąć pod uwagę:
- App Service
- Cosmos DB
- Azure AD

Wynikiem zadania powinny być:
- diagram
- wycena rozwiązania
- dyskusja

**Diagramy i wyceny wrzucajcie tutaj**: https://bit.ly/3fO6D50

Jeśli chcesz nauczyć się jak tworzyć architekturę chmurową dołącz do nas!
Każde spotkanie to nowe architektoniczne zadanie, które pozwala nam poznać dany fragment Azure - omawiamy nasze propozycje i w ten sposób zdobywamy wiedzę!

✨✨✨ Najbliższe spotkanie: **10/12/2020 godzina 20:00**

💻 Link do spotkania: https://bit.ly/36jLuwk
📅 Kalendarz: https://bit.ly/3loHI9n
📖 Poprzednie spotkania (wraz z nagraniami): https://bit.ly/3fO6D50

Masz pytanie? Napisz do mnie lub @maciejgos !

**Comment by aryll** [comment]
Cześć 😄 

Dostępna jest już **notatka z listopadowego spotkania**, gdzie znajdziecie:
- nagranie i prezentację
- linki, które omawialiśmy
- linki ogólno-architektoniczne w tym **katalog przykładowych architektur** stworzony przez Microsoft
- linki, które mogą przydać się do następnego spotkania (w tym application flows)

https://github.com/AzureCommunityPL/group/blob/master/meetings/azure-solutions-architect/azure-solutions-architect-qa-2020-11-26.md

Mamy też **notatkę na następne spotkanie** - tutaj dorzucajcie punkty do **agendy**, które chcielibyście poruszyć
https://github.com/AzureCommunityPL/group/blob/master/meetings/azure-solutions-architect/azure-solutions-architect-qa-2020-12-10.md

**Schematy architektur i wyceny zadania** wrzucajcie natomiast tutaj:
https://github.com/AzureCommunityPL/group/tree/master/meetings/azure-solutions-architect/zadania-2020-12-10

Jeśli macie jakiekolwiek pytania lub potrzebujecie pomocy śmiało pytajcie na priv do mnie lub @maciejgos

**Comment by maciejgos** [comment]
Cześć kolejne spotkanie już za nami. 

Do 24.12 czekamy na wasze propozycje o czym chcielibyście usłyszeć na kolejnym spotkaniu.

Wasze propozycje wrzucajcie pod tym postem
